projects/angular-split/src/lib/component/split.component.ts

```diff
@@ -521,7 +523,38 @@ export class SplitComponent implements AfterViewInit, OnDestroy {
     }
   }
 
-  public startDragging(event: MouseEvent | TouchEvent, gutterOrder: number, gutterNum: number): void {
+  public startKeyboardDrag(event: KeyboardEvent, gutterOrder: number, gutterNum: number) {
+    switch (event.key) {
+      case 'ArrowLeft':
+      case 'ArrowRight':
+      case 'ArrowUp':
+      case 'ArrowDown':
+          break
```

Comment:
Comment from internal team member:
"Looks like you’re still redundantly switching on event key within the same method.  I suggest that you combine the offset setting with the arrowkey validation to avoid duplicating this check."

Reply:
Removed check highlighted above. Leave it to "getKeyboardEndpoint" to do the check